modules/microshift-auto-recovery-example-rpm-systems.adoc

```diff
@@ -4,18 +4,15 @@
 
 :_mod-docs-content-type: PROCEDURE
 [id="microshift-automation-example-rpm-systems_{context}"]
-= Automating the integration process with systemd for RPM systems
+= Using auto-recovery in RPM systems
 
-[NOTE]
-====
-When the `microshift.service` enters a failed state, `systemd` starts the `microshift-auto-recovery.service` unit. This unit executes the `auto-recovery` restore process and restarts {microshift-short}.
-====
+When {microshift-short} enters a failed state, the systemd service starts the `microshift-auto-recovery.service` unit. This unit executes the `auto-recovery` restore process.
 
-As a use case, consider the following example situation in which you want to automate the `auto-recovery` process for RPM systems that use systemd.
+As a use case, consider the following example situation in which you want to automate the auto-recovery process for RPM systems that use the systemd service.
 
 .Procedure
 
-. Create a directory for the `microshift.service` by running the following command:
+. Create a directory for the `microshift.service` ID by running the following command:
```

Comment:
Ha, so in full sentence I'd say something different.
```suggestion
. Create a directory for the `microshift.service` systemd service unit by running the following command:
```

We could also drop `.service`:
```
. Create a directory for the `microshift` systemd service by running the following command:
```